cli/cortx-prvsnr-cli.spec

```diff
@@ -20,6 +20,28 @@ CORTX Provisioner Command line interface. Provides utilities to deploy CORTX Obj
 %prep
 %setup -n %{name}-%{version}-%{_cortx_prvsnr_git_ver}
 
+/bin/cat <<EOL > ssh_config
+Host srvnode-1 srvnode-1.colo.seagate.com
+    HostName srvnode-1.colo.seagate.com
+    User root
+    UserKnownHostsFile /dev/null
+    StrictHostKeyChecking no
+    IdentityFile /root/.ssh/id_rsa_prvsnr
+    IdentitiesOnly yes
+    LogLevel ERROR
+    BatchMode yes
+
+Host srvnode-2 srvnode-2.colo.seagate.com
+    HostName srvnode-2.colo.seagate.com
```

Comment:
no on hardcoding "colo.seagate.com"

Reply:
This is just a template.. we would be updating it with hostnames in later stages of provisioning.

Reply:
@sradhanand-pati change this to:
`Host srvnode-1 srvnode-1.localdomain`